src/tests/utils/ApiConstants.ts

```diff
@@ -6,4 +6,5 @@ export const constants = {
 	DEV_BAV_TEST_HARNESS_URL: process.env.DEV_BAV_TEST_HARNESS_URL,
 	DEV_BAV_SESSION_TABLE_NAME: "session-bav-cri-ddb",
 	DEV_BAV_PERSONAL_IDENTITY_TABLE_NAME: "person-identity-bav-cri-ddb",
+	PERSON_INFO_PRIVATE_KEY: process.env.PERSON_INFO_PRIVATE_KEY,
```

Comment:
Ah ok yeah so we're going to have to put the private key into the run-tests script if we do it like this aren't we. Guess it's an open question for the new year as to whether that's okay?